ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/part/editor/AddEditorTabMenuFactory.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.ide.part.editor;
+
+/** Factory for creating instances of AddEditorTabMenu */
```

Comment:
Please, use links in javadocs for references to the classes. It allows to ensure that javadoc will be updated during some refactoring session and also links allow easy navigation in the generated javadoc.

 `{@link AddEditorTabMenu}`